cmd/kubelet/app/server.go

```diff
@@ -216,7 +216,7 @@ func NewKubeletServer() *KubeletServer {
 		RootDirectory:                  defaultRootDir,
 		SerializeImagePulls:            true,
 		StreamingConnectionIdleTimeout: 5 * time.Minute,
-		SyncFrequency:                  10 * time.Second,
+		SyncFrequency:                  1 * time.Minute,
```

Comment:
@yujuhong cool, that makes sense to me -- this seems like the kind of thing that could affect some of the E2Es that are outside the merge path -- have you run any of those?